client/src/components/Verejne/Map/AddressDetail/ListRow.js

```diff
@@ -23,24 +57,43 @@ const DetailedInfo = compose(
     (state, {id}) => ({
       data: entityDetailSelector(state, id),
     }),
-    {toggleEntityInfo}
+    {toggleEntityInfo, toggleModalOpen, setEntitySearchFor},
   ),
-  withDataProviders(({id}) => [entityDetailProvider(id)])
+  withDataProviders(({id}) => [entityDetailProvider(id)]),
+  withHandlers({
+    openModalSearch: ({toggleModalOpen, setEntitySearchFor, data}) => (e) =>{
+      if (e.stopPropagation) e.stopPropagation()
```

Comment:
why this ? does anyone set e.stopPropagation elsewhere ?

Reply:
Yes, in EntitySearch - but that is a case of form submit, it should not be needed here.